core-plugins/widgets/Email-postaction.json

```diff
@@ -0,0 +1,83 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "Distinct",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Sender",
+          "name": "sender"
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
can you use `csv` widget type here, that will simplify user error and have better experience.
reference : https://github.com/caskdata/hydrator-plugins/blob/develop/core-plugins/widgets/Cube-batchsink.json#L20